sdk/iot/hub/src/az_iot_hub_client_methods.c

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Can you run the clang formatter on this file?

Reply:
Done. For reference, ALT SHIFT F is for VS code on Windows.
On Linux its CTRL SHIFT I